pom.xml

```diff
@@ -521,12 +521,8 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.19.1</version>
         <configuration>
-          <excludes>
-            <exclude>org/folio/rest/api/*Test.java</exclude>
```

Comment:
This exclusion was in place due to tests being run as part of the suite were also being run outside of the suite.

How is that repeated execution avoided in the new approach?

Reply:
My understanding is that the Inclusion what brought in the *Test.java.
By only including the Suites, the explicit exclusion is not needed.
Or so goes my thinking.

Reply:
Cool, that's much neater.